### PR TITLE
Navigation

### LINKED ISSUE #5: Navigation/Header
- [x] Standard
image
- [x] Short
image
- [x] Fixed
image

### PR DESCRIPTION
**Pull request by mohitdhatrak**
To check nav-fixed working, check deploy preview: https://deploy-preview-22--spark-ui.netlify.app/components/navigation/navigation

# Navigation

image

image

Closes #5

### CODE REVIEW

**Comment by ark2002** [comment]
hey it looks good but i am kinda confused with your usage of rems and ems like some places you used rems while some you have used ems

**Review by ark2002 — APPROVED** [review]
looks good to me

**Comment by mohitdhatrak** [comment]
> hey it looks good but i am kinda confused with your usage of rems and ems like some places you used rems while some you have used ems

@ark2002 did that for padding, gaps mostly. Saw a video on youtube that's why did mostly it won't matter on my website.

File: Components/Navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    .navigation-button:nth-child(1),
+.navigation-short
+    .navigation-right-buttons-container
+    .navigation-button:nth-child(2) {
+    display: none;
+}
+
+.navigation-svg-icons {
+    color: var(--navigation-icon-color);
+    font-size: var(--navigation-icon-size) !important;
```
Comment (xii-vi) [inline Components/Navigation/navigation.css:86]: why have you used !important

Comment (mohitdhatrak) [reply Components/Navigation/navigation.css:86]: Could not do anything as my class could not overwrite google icons font size for some reason so used this only for icons size

**Review by xii-vi — APPROVED** [review]
Looks good to me

**Pull request merged**